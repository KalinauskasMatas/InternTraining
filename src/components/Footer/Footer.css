footer {
  background-color: var(--dark-button-color);
  color: var(--bright-text-color);
  position: fixed;
  bottom: 0;
  left: 0;
  right: 0;
  height: 87px;
  display: flex;
  justify-content: space-between;
  align-items: flex-end;
  box-sizing: border-box;
  padding: 15px 20px;
  font-size: 12px;
}