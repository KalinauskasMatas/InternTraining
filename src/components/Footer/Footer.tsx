import "./Footer.css";

const Footer = () => {
  return (
    <footer>
      <span>Made by: Matas Kalinauskas</span>
      <span>2023</span>
    </footer>
  );
};

export default Footer;
