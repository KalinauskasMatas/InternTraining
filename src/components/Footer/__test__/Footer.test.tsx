import Footer from "../Footer";

import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom";

import renderer from "react-test-renderer";

afterEach(cleanup);

it("renders without crashing", () => {
  render(<Footer />);
});

it("matches snapshot", () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
