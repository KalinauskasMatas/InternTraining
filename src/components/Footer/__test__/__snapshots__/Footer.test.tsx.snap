// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<footer>
  <span>
    Made by: Matas Kalinauskas
  </span>
  <span>
    2023
  </span>
</footer>
`;
